### PR TITLE
Fix build issue

### PR DESCRIPTION
**Pull request by kaftand**

### CODE REVIEW

File: UpdateVersion.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+with open("VERSION", mode="r+") as f:
+  version = f.read().split(".")
+  version[-1] = str(int(version[-1]) + 1)
+  f.seek(0)
+  f.write(".".join(version))
```
Comment (paulkaefer) [inline UpdateVersion.py:5]: do we want to have a standard of spaces vs. tabs, and if so, # of spaces? I prefer four

also might not cause issues for us, but it's [considered good style](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) to have a newline at the end of every file

File: pyproject.toml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools","cython", "numpy","scipy"]
\ No newline at end of file
+requires = ["setuptools","cython", "numpy==1.19.2","scipy"]
```
Comment (paulkaefer) [inline pyproject.toml:3]: should we do >= instead of exact version?

File: UpdateVersion.py (hunk 1 above)
Comment (kaftand) [reply UpdateVersion.py:5]: yeah, you're right. I prefer 2, but pep says 4. argh.

File: pyproject.toml (hunk 2 above)
Comment (kaftand) [reply pyproject.toml:3]: Right now the latest version of numpy dies on windows. Probably eventually we want >=, but now we don't have much of a choice

**Pull request merged**